PublicMapping/districtbuilder-classic
# Add email button to home and share plan page

## Overview

Add email button to home and share plan page. The button is a mailto link which sends to the `MAILER_USER` in the `.env` file, has a subject line that corresponds to the page title, and the body with the page URL. 

### Checklist


### Demo

<img width="598" alt="screen shot 2018-04-27 at 4 03 03 pm" src="https://user-images.githubusercontent.com/3959096/39382731-7c2579cc-4a34-11e8-9943-d7ab65feaee4.png">

<img width="624" alt="screen shot 2018-04-27 at 4 01 03 pm" src="https://user-images.githubusercontent.com/3959096/39382736-7f6b6740-4a34-11e8-9878-c72d2536f704.png">

## Testing Instructions

 * Go to home page and check if the email button exists. Check the mailto link to make sure it has the correct email (same as `MAILER_USER` in the `.env` file), subject, and link in the body.
 * Log in and go to share page for a plan. Check the mailto link to make sure it has the correct email (same as `MAILER_USER` in the `.env` file), subject, and link in the body.
